src/plot/plcontourplot.cpp

```diff
@@ -250,16 +261,17 @@ void wxPLContourPlot::Draw(wxPLOutputDevice &dc, const wxPLDeviceMapping &map) {
         dc.NoPen();
         // background set to min
         // assume RebuildMask has been called
-        wxColor bgc(m_cmap->ColourForValue(m_zMin));
+        //wxColor bgc(m_cmap->ColourForValue(m_zMin));
+        wxColor bgc(*wxLIGHT_GREY);
```

Comment:
Maybe this should be a defined value like CONTOUR_BG or COUNTOUR_NAN_COLOR for future maintenance - what do you think?